src/components/Badges/Accordion.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import {
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Typography,
+  Stack,
+} from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import AccordionPill from './AccordionPill';
+
+export default function CustomAccordion({
+  children,
+  title,
+}: {
+  title: string;
+  children?: React.ReactNode;
+}) {
+  // State for the expanded accordion panel
+  const [expanded, setExpanded] = useState<string | false>(false);
```

Comment:
why not keep this as boolean?
```suggestion
  const [isExpanded, setIsExpanded] = useState<false>(false);
```

Reply:
I don't see a reason to support a string type for this state